src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -142,17 +142,73 @@ public int getSampleRate() {
 	}
 
 	/**
+	 * @return True if the audio codec is 3GA.
+	 */
+	public boolean is3GA() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.THREEGA);
+	}
+
+	/**
+	 * @return True if the audio codec is AAC.
+	 */
+	public boolean isAAC() {
+		return getCodecA() != null && (getCodecA().equalsIgnoreCase(FormatConfiguration.AAC) || getCodecA().equalsIgnoreCase(FormatConfiguration.AAC_HE));
+	}
+
+	/**
+	 * @return True if the audio codec is AAC LC.
+	 */
+	public boolean isAACLC() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.AAC);
+	}
+
+	/**
 	 * @return True if the audio codec is AC-3.
 	 */
 	public boolean isAC3() {
 		return getCodecA() != null && (getCodecA().equalsIgnoreCase("ac3") || getCodecA().equalsIgnoreCase("a52") || getCodecA().equalsIgnoreCase("liba52"));
 	}
 
 	/**
-	 * @return True if the audio codec is TrueHD.
+	 * @return True if the audio codec is ADPCM.
 	 */
-	public boolean isTrueHD() {
-		return getCodecA() != null && getCodecA().equalsIgnoreCase("truehd");
+	public boolean isADPCM() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.ADPCM);
+	}
+
+	/**
+	 * @return True if the audio codec is AIFF.
+	 */
+	public boolean isAIFF() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.AIFF);
+	}
+
+	/**
+	 * @return True if the audio codec is ALAC.
+	 */
+	public boolean isALAC() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.ALAC);
+	}
+
+	/**
+	 * @return True if the audio codec is ATRAC.
+	 */
+	public boolean isATRAC() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.ATRAC);
+	}
+
+	/**
+	 * @return True if the audio codec is Cook.
+	 */
+	public boolean isCook() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.COOK);
+	}
+
+	/**
+	 * @return True if the audio codec is DSD Audio.
+	 */
+	public boolean isDSDAudio() {
+		return getCodecA() != null && (getCodecA().equalsIgnoreCase(FormatConfiguration.DSDAudio));
```

Comment:
@Sami32 I know, that issue has been there for years - he wants money for just about anything it seems.